Thermodynamics.pyx

```diff
@@ -56,7 +56,7 @@ cdef class ClausiusClapeyron:
             Tmin = namelist['ClausiusClapeyron']['temperature_min']
         except:
             Par.root_print('Clasius-Clayperon lookup table temperature_min not '
-                           'given in name list taking default of 100.15 K')
+                           'given in name list taking default of 180 K')
             Tmin = 100.15
```

Comment:
Looks like in the print we say that the default is 180 but then we set it to 100.15?